website/content/docs/uploadcare.md

```diff
@@ -46,7 +46,7 @@ using the image or file widgets.
 ## Configuring the Uploadcare Widget
 
 The Uploadcare widget can be configured with settings that are outlined [in their
-docs](https://uploadcare.com/docs/uploads/widget/config/). The widget itself accepts configration
+docs](https://uploadcare.com/docs/file_uploads/widget/options/). The widget itself accepts configration
```

Comment:
I think the equivalent would be:
https://uploadcare.com/docs/file_uploads/widget/config/

What do you think?

Reply:
Uploadcare redirects to the url in this PR, and that URL does provide the list of configuration options, which is the intended target here.